src/components/Modal/index.js

```diff
@@ -65,46 +66,55 @@ const CancelButtonContainer = styled(Row)`
   padding-right: ${spacing.moderate};
 `;
 
-const Modal = ({ children }) => (
+const Modal = ({ children, container, content, ...props }) => (
   <BackdropConsumer>
     {backdropValue => (
       <ModalContainer
         small={4}
         role="dialog"
         aria-modal
         innerRef={backdropValue ? backdropValue.childRef : null}
+        style={{ ...container }}
       >
-        <CancelButtonContainer>
-          <ItemContainerConsumer>
-            {value => (
-              <IconButton
-                className="button--close"
-                size={45}
-                aria-label="Close Modal"
-                role="button"
-                onClick={value ? value.onCloseRequest : () => {}}
-              >
-                <CrossIcon
-                  size={12}
-                  style={{ pointerEvent: "none" }}
-                  color={colors.onyx.base}
-                />
-              </IconButton>
-            )}
-          </ItemContainerConsumer>
-        </CancelButtonContainer>
-        <ModalContent>{children}</ModalContent>
+        {!props.noCloseIcon && (
```

Comment:
Can you please make this a `boolean` prop which causes the close icon to render when it is `true`, so it is consistent with other components? E.g.`withCloseIcon`. See [`BottomSheet`](https://github.com/ticketmaster/aurora/blob/master/src/components/BottomSheet/index.js#L60) for an example (`withCancelBtn`).

Reply:
sure thing